icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/ListFormatterTest.java

```diff
@@ -292,4 +293,90 @@ public void TestCreateStyled() {
             assertEquals(message, expected, result);
         }
     }
+
+    @Test
+    public void TestSpanish() {
+        String[] locales = { "es", "es_419", "es_PY", "es_DO" };
```

Comment:
As in C++ -- why not combine the spanish & hebrew tests into one function with more test cases?
Less code to review & maintain...

Reply:
ok